core/Tracker/Request.php

```diff
@@ -294,11 +315,11 @@ public function getParam($name)
             // other
             'bots'         => array(0, 'int'),
             'dp'           => array(0, 'int'),
-            'rec'          => array(false, 'int'),
+            'rec'          => array(0, 'int'),
```

Comment:
I changed the default values to the actual default values that are used by `Common::getRequestVar` as this method casts `int` and `string` default values to the requested `type`